Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version:5.0.1
+
+//
+//  Copyright 2020 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import PackageDescription
+
+let package = Package(
+	name: "Stagehand",
+	platforms: [
+		.iOS(.v10),
+	],
+	products: [
+		.library(
+			name: "Stagehand",
+			targets: ["Stagehand"]
+		),
+	],
+	targets: [
+		.target(
```

Comment:
Consider adding your test target to this list as well.

Reply:
This is a bit more complex since some of the tests depend on StagehandTesting, which isn't ready to move to SPM. I'll look into breaking up the tests, but going to merge this without the test target to unblock getting SPM support for Stagehand.